src/commands/eval.ts

```diff
@@ -530,6 +531,16 @@ export async function doEval(
     if (testSuite.redteam) {
       showRedteamProviderLabelMissingWarning(testSuite);
     }
+
+    // Clean up any WebSocket connections
+    if (testSuite.providers.length > 0) {
+      for (const provider of testSuite.providers) {
+        if (provider instanceof OpenAiRealtimeProvider) {
+          provider.cleanup();
```

Comment:
This is a great idea. We can add it as an optional method to api provider 

```suggestion
          provider?.cleanup();
```

Reply:
See https://github.com/promptfoo/promptfoo/blob/182e9af16a30202ef131cbec5a83ed4244f0dfdb/src/types/providers.ts#L75